pkgs/applications/misc/heimer/default.nix

```diff
@@ -0,0 +1,23 @@
+{ mkDerivation, lib, fetchFromGitHub, cmake, qttools, qtbase }:
+
+mkDerivation rec {
+  pname = "heimer";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "juzzlin";
+    repo = pname;
+    rev = "refs/tags/${version}";
```

Comment:
Does `rev = version` doesn't work?

Reply:
(see comment below for response, sorry-- but in short, pushed commit fixing to just be `rev = version`)